edx_exams/apps/router/interop.py

```diff
@@ -23,10 +32,67 @@ def register_exams(course_id, exam_list):
         response = make_request('PATCH', url, client, json=exam_list)
     except HTTPError as e:
         response = e.response
-    return response
+
+    response_data = _get_json_data(response)
+    if response.status_code != status.HTTP_200_OK:
+        log.error(
+            f'Failed to publish exams for course_id {course_id} '
+            f'got status={response.status_code} content={response.content}'
+        )
+
+    return response_data, response.status_code
+
+
+def get_student_exam_attempt_data(course_id, content_id, lms_user_id):
+    """
+    Get student exam attempt data from the legacy proctoring service
+    """
+    content_id_url_safe = quote_plus(content_id)    # because this goes in the query string
+    path = LMS_PROCTORED_EXAM_ATTEMPT_DATA_API_TPL.format(course_id, content_id_url_safe, lms_user_id)
+    url = _proctoring_api_url(path)
+    client = get_client(settings.LMS_ROOT_URL)
+    try:
+        response = make_request('GET', url, client)
+    except HTTPError as e:
+        response = e.response
+
+    response_data = _get_json_data(response)
```

Comment:
I don't know if this is logging overkill, but it might be helpful to log that exams is getting the attempt data from the legacy service (just for debugging purposes in the future).

I'm not sure what edx-proctoring logs though

Reply:
oh there's a log for this up in the middleware that says the request will be forwarded.

Reply:
whoops sorry for missing that. sounds good 👍